clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/api/KubernetesApiAdaptor.groovy

```diff
@@ -148,18 +149,24 @@ class KubernetesApiAdaptor {
     }
   }
 
-  List<ReplicationController> getReplicationControllers(String namespace) {
-    atomicWrapper("Get Replication Controllers", namespace) { KubernetesClient client ->
-      client.replicationControllers().inNamespace(namespace).list().items
+  List<ReplicaSet> getReplicaSets(String namespace) {
+    atomicWrapper("Get Replica Sets", namespace) { KubernetesClient client ->
+      client.extensions().replicaSets().inNamespace(namespace).list().items
     }
   }
 
-  List<Pod> getReplicationControllerPods(String namespace, String replicationControllerName) {
-    atomicWrapper("Get Replication Controller Pods for $replicationControllerName", namespace) { KubernetesClient client ->
+  List<Pod> getReplicaSetPods(String namespace, String replicationControllerName) {
```

Comment:
Should the other replication controller names here change as well (e.g. `replicationControllerName`)?